README.md

```diff
@@ -10,10 +15,124 @@ State-driven, resumable download API.
 | Android  | ✓         |
 | iOS      | ✓         |
 
-## Contributing
+## Installation
+
+Note: These steps are an interim workaround until the plugin can be published to npm/crates.io.
+
+### Rust
+
+Add the `tauri-plugin-download` crate to your `Cargo.toml`:
+
+```toml
+[dependencies]
+tauri-plugin-download = { git = "https://github.com/silvermine/tauri-plugin-download.git" }
+```
+
+### TypeScript
+
+Install the TypeScript bindings via npm:
+
+```bash
+npm install github:@silvermine/tauri-plugin-download
+```
+
+## Usage
+
+### Rust
+
+Initialize the plugin in your `tauri::Builder`:
+
+```rust
+fn main() {
+   tauri::Builder::default()
+      .plugin(tauri_plugin_download::init())
+      .run(tauri::generate_context!())
+      .expect("error while running tauri application");
+}
+```
+
+### TypeScript
+
+#### Create a download
+
+```ts
+import { create } from "tauri-plugin-download";
+
+async function createDownload() {
+   const key = "file.zip";
+   const url = "https://example.com/file.zip";
+   const path = "/path/to/save/file.zip";
+
+   const download = await create(key, url, path);
+   console.log(`Created '${download.key}':${download.url}`)
```

Comment:
Nit: Could you update the code samples so that they pass our ESLint linting if they were copy/pasted somewhere?

```suggestion
   const key = 'file.zip',
         url = 'https://example.com/file.zip',
         path = '/path/to/save/file.zip';

   const download = await create(key, url, path);

   console.log(`Created '${download.key}':${download.url}`)
```

* Double quotes to single quotes
* Newline after `const` declaration
* Combine `const` / `let` declarations where possible/readable